trackr-app/app/src/main/java/com/trackr/trackr_app/entities/User.kt

```diff
@@ -0,0 +1,9 @@
+package com.trackr.trackr_app.entities
+
+import java.util.UUID
+
+class User (
```

Comment:
A [dataclass](https://kotlinlang.org/docs/data-classes.html) can be used instead of a user if all this class needs to do is store data.

Reply:
Let's keep it a regular class for now and re-evaluate later.